.gitattributes

```diff
@@ -1 +1,3 @@
 jupyterlab/staging/yarn.js binary
+# SCM syntax highlighting
+pixi.lock linguist-language=YAML linguist-generated=true
```

Comment:
This would only matter if we did add a lockfile to repo, right?

Reply:
Right, this is more for convenience for now, as running `pixi init` would otherwise modify the `.gitattributes` file to add that line automatically.

Reply:
Makes sense then!